### PR TITLE
modal popup window

### PR DESCRIPTION
**Pull request by DevKaranJ**
This pull request adds the implementation of a details popup window for each project in my portfolio website. The goal of this milestone was to use basic JavaScript syntax, manipulate DOM elements, handle JavaScript events, and create a UI based on a Figma design.

### CODE REVIEW

File: modal.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  font-weight: 600;
+  line-height: 16px;
+  color: #3c3a39;
+  padding: 5px 10px;
+}
+
+/* .modalProjectImg {
+  width: 100%;
+  padding: 20px 0;
+} */
```
Comment (rica213) [inline modal.css]: - [x] Kindly remove the commented-out code to keep your code source clean 🧹

File: script.js [hunk 2]
```diff
@@ -63,4 +63,99 @@ function loadFormData() {
...
+  },
+  {
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+];
```
Comment (rica213) [inline script.js]: - [ ] Please create distinct data for your projects to distinguish the change during testing 📓

File: script.js [hunk 3]
```diff
@@ -63,4 +63,99 @@ function loadFormData() {
...
+  });
+
+  document.getElementById('modalLiveVersionLink').href = projectInfo.live_version;
+  document.getElementById('modalSourceLink').href = projectInfo.link_source;
+
+  const modal = document.getElementById('modal');
+  modal.style.display = 'block';
+}
+
+openModal(0);
```
Comment (rica213) [inline script.js]: - [x] Great job implementing the modal. However, no matter which project I want to check I will always get the first project shown 🔢 Kindly fix that detail 🙏🏽

File: script.js [hunk 4]
```diff
@@ -63,4 +63,99 @@ function loadFormData() {
...
+    name: 'Multi-Post Stories',
+    cardDetail: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    skills: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/ProjectImg/modalProject.png',
+    live_version: '#',
+    link_source: '#',
+  },
+];
+
+function openModal() {
```
Comment (rica213) [inline script.js]: - [x] When I refresh the browser, without clicking the button `see project`, a modal is displaying 🖼️ That is not a behavior we were expecting. Kindly remedy that detail 🧄

File: modal.css [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  padding: 0 10px;
+}
```
Comment (rica213) [inline modal.css:12]: - [x] Kindly fix the `scrolling` behavior of your modal. For example, in the mobile, I have a design similar to the screenshot below 👇🏽  The `live version` and `source code` buttons are not seen 👀 The issue is when I scroll down to access them, only the `body` element is scrolling.

Screenshot from 2023-07-29 11-37-29

File: modal.css [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+  border-radius: 5px;
+  font-weight: bold;
+}
+
+/* Modal desktop */
+@media screen and (min-width: 768px) {
+  .desktopModalDesign {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
```
Comment (rica213) [inline modal.css:96]: - [x]  The buttons `live version` and `source code` on the desktop design are not aligned properly 🔢 Kindly refer to the screenshot below 👇🏽 and fix the alignment according to the Figma design ❤️ 

Screenshot from 2023-07-29 11-43-05

**Review by rica213 — CHANGES_REQUESTED** [review]
# 🟡 STATUS: REQUIRED CHANGES ♻️ 
Hi Karan, 
My name is Larissa. Your code reviewer 🥰

Good job so far!
However, there are some issues that I want you to address 🤗  Refer them in the comment section below 👇🏽 



Happy coding!

## Highlights
Good Design ✔️
Professional Documentation ✔️
No Linter errors ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


# Cheers, and Happy coding 🖥️

## If anything is not 100% clear, kindly mention me in your comment and I will reply ASAP 🤞

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @DevKaranJ   , 

### Highlights ✨ 

- Thanks for making the requested changes 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**